src/components/game/Board.js

```diff
@@ -306,7 +323,7 @@ Board.propTypes = {
   whiteTeamWord: PropTypes.string.isRequired,
   blueTeamWord: PropTypes.string.isRequired,
   role: PropTypes.string.isRequired,
-  yourTurn: PropTypes.bool.isRequired,
+  yourTurn: PropTypes.string.isRequired,
```

Comment:
should yourTurn be a string or a bool?

Reply:
I just looked at the parent component and yeah it should be a bool

Reply:
maybe you can make the name of yourTurn be "canDraw" so that it makes more sense with the "drawing" boolean

Reply:
ya you're right I think canDraw is a better name

Reply:
I agree- canDraw is a better name- more indicative of what the variable tracks. I think it should be a boolean; it is more intuitive to have the ability to draw be determined by a true or false value, rather than having it as a string that needs to be parsed.